LibreNMS/__init__.py

```diff
@@ -371,10 +525,20 @@ def put(self, item):
         self._redis.zadd(self.key, {item: time()}, nx=True)
 
     def get(self, block=True, timeout=None):
-        if block:
-            item = self._redis.bzpopmin(self.key, timeout=timeout)
-        else:
-            item = self._redis.zpopmin(self.key)
+        try:
+            if block:
+                item = self._redis.bzpopmin(self.key, timeout=timeout)
+            else:
+                item = self._redis.zpopmin(self.key)
+        # Unfortunately we cannot use _redis.exceptions.ResponseError Exception here
+        # Since it would trigger another exception in queuemanager
+        except Exception as e:
```

Comment:
I think you might catch other exceptions here and confuse the user with your specific message.  Can we make this more narrow?

Reply:
I honestly tried to catch a more narrow exception, hence the comment.
How about I change the exception error message to the following ?
```
"BZPOPMIN/ZPOPMIN command failed: {}\nNote that redis >= 5.0 is required.".format(e)
```

Reply:
Perhaps just send the message of the original exception and append the note :)

Reply:
Just commited, should be good